### ISSUE TITLE
feature request: shuffle by artist

### ISSUE THREAD

**Issue by BobCostanzo** [HardcodeCoder/PulseMusic#85]
The idea is simply: when you visit the Artist page, a button or other context menu is available to shuffle/play all songs from all albums by the selected artist. 

Thank you for this amazing app. Best music player on android, bar none.

**Comment by jetrosut** [comment]
I agree, this would make a great feature. The shuffle button could work differently depending on where you are in the menus. When you are playing a song from a list of all the music you have, the shuffle could include all your music but if you are in an artist's or an album's page and you are playing the music there, it should shuffle songs only within the folder/group you are viewing.
